routes/users.js

```diff
@@ -5,59 +5,7 @@ const usersController = require('../controllers/usersController')
 const userValidator = require('../middlewares/validators/user')
 
 /**
- * @swagger
- * /users:
- *   post:
- *     summary: Create new user with provided data.
- *
- *     requestBody:
- *       description: User data
- *       content:
- *         application/json:
- *           schema:
- *             $ref: '#/components/schemas/users'
- *
- *     tags:
- *       - Users
- *     security:
- *       - bearerAuth: []
- *     responses:
- *       200:
- *         description: User details
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/users'
- *
- *       401:
- *         description: unAuthorized
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/unAuthorized'
- *       403:
- *         description: forbidden
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/forbidden'
- *       404:
- *         description: notFound
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/notFound'
- *       503:
- *         description: serverUnavailable
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/serverUnavailable'
- */
-router.post('/', authenticate, userValidator.createUser, usersController.addNewUser)
```

Comment:
@JatinAgrawal94 , can we also remove the tests for this route?

Reply:
This route allows us creating a new user, making a post request. It needs to be kept.

Reply:
Github OAuth takes care of creating users. Please read the issue carefully: #161 

Any random person can create any dummy account with this route.